src/sentry/static/sentry/app/views/availableWorkDetails/index.jsx

```diff
@@ -0,0 +1,131 @@
+import PropTypes from 'prop-types';
```

Comment:
This must be the view showing when user has clicked on a single workstep to see what samples in it?

Reply:
Right. Technically, it will list work units that are in it, but the user sees samples.